File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        n=len(nums)
        t=[]
        for i in range(n):
            for j in range(0,i+1):
                if i!=j:
                    if nums[i]+nums[j]==target:
                        t=[i,j]
        return t
solution=Solution()
nums=[2,7,11,15]
res=solution.twoSum(nums,9)

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        int[] result = new int[2];

        for (int i = 0; i < n; i++) {
            for (int j = 0; j <= i; j++) {
                if (i != j) {
                    if (nums[i] + nums[j] == target) {
                        result[0] = i;
                        result[1] = j;
                        return result;
                    }
                }
            }
        }
        return result;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] nums = {2, 7, 11, 15};
        int target = 9;
        int[] res = solution.twoSum(nums, target);
        System.out.println(Arrays.toString(res));
    }
}